http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -527,6 +531,51 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
     }
   }
 
+  private[monitor] def createGalaxyPostgresDiskOnlyInGoogle(project: GoogleProject,
+                                                            zone: ZoneName,
+                                                            appName: AppName,
+                                                            namespaceName: NamespaceName)(
+    implicit ev: ApplicativeAsk[F, AppContext]
+  ): F[Unit] = {
+    // TODO: remove post-alpha release of Galaxy. For pre-alpha we are only creating the postgress disk in Google since we are not supporting persistence
+    // see: https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/859406337/2020-10-02+Galaxy+disk+attachment+pre+post+alpha+release
+    val create = for {
+      ctx <- ev.ask
+      _ <- logger.info(s"Beginning postgres disk creation for app ${appName.value} | trace id: ${ctx.traceId}")
+      operation <- googleDiskService.createDisk(project,
+                                                zone,
+                                                gkeInterp.createGalaxyPostresDisk(project, zone, namespaceName))
+      whenDone = logger.info(
+        s"Completed postgres disk creation for app ${appName.value} in project ${project.value} | trace id: ${ctx.traceId}"
+      )
+      _ <- computePollOperation.pollZoneOperation(
+        project,
+        zone,
+        OperationName(operation.getName),
+        config.persistentDiskMonitorConfig.create.interval,
+        config.persistentDiskMonitorConfig.create.maxAttempts,
+        None
+      )(
+        whenDone,
+        F.raiseError(
+          new TimeoutException(
+            s"Failed to create Galaxy postgres disk in a timely manner. Project: ${project.value}, AppName: ${appName.value}"
+          )
+        ),
+        F.unit
+      )
+    } yield ()
+
+    create.onError {
+      case e =>
+        for {
+          _ <- logger.error(e)(
```

Comment:
I think this will log the same error twice since the main error handler also logs error https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala#L113...probably better to create a `PostgresDiskError` similar to `PubsubKubernetesError`....that away you can remove `onError`, and it'll be logged in the main error handler

Reply:
i'm not sure it'll log it twice, we do the same for normal disk creation (in google + in our db): https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala#L520